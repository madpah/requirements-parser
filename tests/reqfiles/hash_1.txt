packaging==21.3 \
    --hash=sha256:ef103e05f519cdc783ae24ea4e2e0f508a9c99b2d4969652eed6a2e1ea5bd522 \
    --hash=sha256:ef103e05f519cdc783ae24ea4e2e0f508a9c99b2d4969652eed6a2e1ea5bd522
    # testing
packaging==21.3 \
    --hash=sha256:ef103e05f519cdc783ae24ea4e2e0f508a9c99b2d4969652eed6a2e1ea5bd522 \
    --hash=sha256:ef103e05f519cdc783ae24ea4e2e0f508a9c99b2d4969652eed6a2e1ea5bd522
