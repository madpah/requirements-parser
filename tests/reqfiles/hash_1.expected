[
 {
  "specifier": true,
  "local_file": false,
  "name": "packaging",
  "editable": false,
  "subdirectory": null,
  "uri": null,
  "extras": [],
  "vcs": null,
  "path": null,
  "line":  "packaging==21.3",
  "hash_name": null,
  "hash": null,
  "specs": [
   [
    "==",
    "21.3"
   ]
  ],
  "revision": null
 },
 {
  "specifier": true,
  "local_file": false,
  "name": "packaging",
  "editable": false,
  "subdirectory": null,
  "uri": null,
  "extras": [],
  "vcs": null,
  "path": null,
  "line":  "packaging==21.3",
  "hash_name": null,
  "hash": null,
  "specs": [
   [
    "==",
    "21.3"
   ]
  ],
  "revision": null
 }
]
